ntoskrnl/CMakeLists.txt

```diff
@@ -12,6 +12,11 @@ set(NTKRNLMP_ASM_SOURCE ${ASM_SOURCE})
 spec2def(ntoskrnl.exe ntoskrnl.spec ADD_IMPORTLIB)
 add_asm_files(ntoskrnl_asm ${NTOSKRNL_ASM_SOURCE})
 
+if (GCC AND CLANG)
+    #Clang optimises strcmp calls to memcmp.
```

Comment:
```suggestion
    # Clang optimises strcmp calls to memcmp.
```

Reply:
I'm not sure this sort of optimization is correct, to begin with.

Reply:
It doesn't always do it. Mainly when one of the two comparands is known at compile time.